audio.js

```diff
@@ -1,12 +1,225 @@
+                keySounds(119, "Heater 2");
+                keySounds(101, "Heater 3");
+                keySounds(97, "Heater 4");
+                keySounds(115, "Clap");
+                keySounds(100, "Open HH");
+                keySounds(122, "Kick Hat");
+                keySounds(120, "Kick");
+                keySounds(99, "Closed HH");
+            });
+        } else {
+            $(document).keypress(function (event) {
+                // console.log(event.keyCode);
+                keySounds(113, "Chord 1");
+                keySounds(119, "Chord 2");
+                keySounds(101, "Chord 3");
+                keySounds(97, "Shaker");
+                keySounds(115, "Open H");
+                keySounds(100, "Closed H");
+                keySounds(122, "Punchy Kick");
+                keySounds(120, "Side Stick");
+                keySounds(99, "Snare");
+            });
+        }
+
+        bank.addEventListener('input', () => {
+            // console.log(bank.value);
+            // console.log(toggle.value);
+            switch (bank.value) {
+                case '0':
+                    // console.log('bank.value == 0');
+                    if (toggle.value == '1') {
+                        $(document).keypress(function (event) {
+                            // console.log(event.keyCode);
+                            keySounds(113, "Heater 1");
+                            keySounds(119, "Heater 2");
+                            keySounds(101, "Heater 3");
+                            keySounds(97, "Heater 4");
+                            keySounds(115, "Clap");
+                            keySounds(100, "Open HH");
+                            keySounds(122, "Kick Hat");
+                            keySounds(120, "Kick");
+                            keySounds(99, "Closed HH");
+                        });
+                    }
+                    break;
+                case '1':
+                    // console.log('bank.value == 1');
+                    if (toggle.value == '1') {
+                        $(document).keypress(function (event) {
+                            // console.log(event.keyCode);
+                            keySounds(113, "Chord 1");
+                            keySounds(119, "Chord 2");
+                            keySounds(101, "Chord 3");
+                            keySounds(97, "Shaker");
+                            keySounds(115, "Open H");
+                            keySounds(100, "Closed H");
+                            keySounds(122, "Punchy Kick");
+                            keySounds(120, "Side Stick");
+                            keySounds(99, "Snare");
+                        });
+                    }
+                    break;
+                default:
+                    console.log('switch');
+            }
+        });
+
+        $(".playback").removeAttr('disabled'); //audio on
+
+        $(".playback").click(function (e) {
```

Comment:
This is what I was talking about before, at line 18. This function can be moved to the top level of the file and doesn't need to be async. See the current audio.js file on our master branch.

Reply:
keySounds function lives inside the toggle event listener aka power on or off, so that keySounds are silenced when the power is off.